src/components/BrowserSupportTable/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styles from './styles.module.css';
+import { headers, browserInfo } from './constants';
+
+const getClassName = value => {
+  if (value === 'No') {
+    return `${styles.backgroundRed} ${styles.crossGradient}`;
+  }
+  if (value === '?') {
+    return styles.backgroundLightGray;
+  }
+  return styles.backgroundGreen;
+};
+
+const Header = ({ name }) => {
+  return (
+    <th className={styles.rotate}>
+      <div>
+        <span>{name}</span>
+      </div>
+    </th>
+  );
+};
+
+const Rows = ({
+  browser,
+  intl,
+  collator,
+  dateTimeFormat,
+  listFormat,
+  locale,
+  numberFormat,
+  pluralRules,
+  relativeTimeFormat,
+  getCanonicalLocales,
+}) => {
+  return (
+    <tr>
+      <td>{browser}</td>
+      <td className={getClassName(intl)}>{intl}</td>
+      <td className={getClassName(collator)}>{collator}</td>
+      <td className={getClassName(dateTimeFormat)}>{dateTimeFormat}</td>
+      <td className={getClassName(listFormat)}>{listFormat}</td>
+      <td className={getClassName(locale)}>{locale}</td>
+      <td className={getClassName(numberFormat)}>{numberFormat}</td>
+      <td className={getClassName(pluralRules)}>{pluralRules}</td>
+      <td className={getClassName(relativeTimeFormat)}>{relativeTimeFormat}</td>
+      <td className={getClassName(getCanonicalLocales)}>
+        {getCanonicalLocales}
+      </td>
+    </tr>
+  );
+};
+
+const BrowserSupportTable = () => {
+  return (
+    <div>
+      <table className={`${styles.backgroundDarkGray} ${styles.table}`}>
+        <thead>
+          <tr>
+            <th className={styles.verticalAlignBottom}>
+              <div>
+                <span>Browser</span>
+              </div>
+            </th>
+            {headers.map(header => (
+              <Header key={header.name} name={header.name} />
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {browserInfo.map(info => (
+            <Rows
```

Comment:
Hi @JoaoJesus94 

I think that we can pass only 1 prop to the Rows component, such as: **info**.

Then, on **Rows** Component, we can simplify a bit, using only the prop passed previously.

An example of Rows Component (not tested 😛)

```
const Rows = (info) => {
  return (
    <tr>
      <td>{info.browser}</td>
      Object.keys(info).filter(column => column !== 'browser').map(col => (
         <td className={getClassName(info[col])}>{info[col]}</td>
      ));
    </tr>
  );
};
```

When calling the Rows component, just use as you have but passing only the info prop.

Maybe something similar with the example below:

```
{browserInfo.map(info => <Rows info={info} />)}
```

This way, we are not hardcoding the way that we're building the rows, and if we need to add any new column, the only thing to do is simply add on `src/components/BrowserSupportTable/constants.js`

Reply:
I'll be using Table from antd on the future